recipe/bld.bat

```diff
@@ -10,7 +10,8 @@ set CXXFLAGS=/MP
 set CFLAGS=/MP
 
 REM Configure Step
-cmake -T "host=x64" -G "%CMAKE_GENERATOR%" ^
+cmake -T "v141,host=x64" -G "Visual Studio 16 2019" ^
+    -D ITK_SKIP_PATH_LENGTH_CHECKS:BOOL=ON ^
```

Comment:
Let's move these changes to a separate PR, as I believe the need a little more investigation to determine if this is the best approach.